pkg/local_object_storage/engine/engine_test.go

```diff
@@ -55,19 +56,25 @@ func testNewShard(t *testing.T, id int) *shard.Shard {
 		shard.WithID(sid),
 		shard.WithLogger(zap.L()),
 		shard.WithBlobStorOptions(
-			blobstor.WithRootPath(fmt.Sprintf("%s.%d.blobstor", t.Name(), id)),
+			blobstor.WithRootPath(fmt.Sprintf("%s/%d.blobstor", t.Name(), id)),
```

Comment:
Maybe `path.Join`? See https://github.com/nspcc-dev/neofs-node/pull/948/commits/11a214bf8925a368f13dd7d761c0a90b30a70012

Reply:
I'll skip this change to avoid conflicts since you've already done it.